spec/index.html

```diff
@@ -167,57 +167,71 @@ <h3>Resources and Statements</h3>
       as the subject.</p>
 
-    <figure id="fig-quoted-triple">
+    <figure id="fig-quoted-graph">
       <a href="quoted-triple.svg">
         <!-- Source for this file is at https://docs.google.com/drawings/d/1I_QxbUgnQXumXzb8c0WNJHIQ-mtRs2S80dDG6i9aOD8 -->
         <img src="quoted-triple.svg"
-             alt="An RDF graph containing a triple that references an unasserted quoted triple (with grey background) as the subject"
+             alt="An RDF graph containing a triple that references an unasserted quoted graph (with grey background) as the subject"
              style="width:70%"
-             aria-describedby="fig-quoted-triple-alt"/>
+             aria-describedby="fig-quoted-graph-alt"/>
       </a>
-      <figcaption id="fig-quoted-triple-alt">
-        An RDF graph containing a triple that references an unasserted quoted triple (with grey background) as the subject.
+      <figcaption id="fig-quoted-graph-alt">
+        An RDF graph containing a triple that references an unasserted quoted graph (with grey background) as the subject.
       </figcaption>
     </figure>
 
-    <p>A variation on the graph shown in <a href="#fig-quoted-triple"></a> can be described
-      where the <a>quoted triple</a> is also <a data-lt="asserted triple">asserted</a>.</p>
+    <p>A variation on the graph shown in <a href="#fig-quoted-graph"></a> can be described
+      where the triples in the <a>quoted graph</a> are also in the <a>default graph</a>.</p>
 
-    <figure id="fig-quoted-asserted-triple">
+    <figure id="fig-quoted-asserted-graph">
       <a href="quoted-asserted-triple.svg">
         <!-- Source for this file is at https://docs.google.com/drawings/d/1aoUEsb07P-nu5rvPRob7O07CRKgnf7tBw2CmRJNuaQ0 -->
         <img src="quoted-asserted-triple.svg"
              alt="An RDF graph containing two triples where one of them contains the other as the subject; hence, that other triple is both quoted and asserted"
              style="width:70%"
-             aria-describedby="fig-quoted-asserted-triple-alt"/>
+             aria-describedby="fig-quoted-asserted-graph-alt"/>
       </a>
-      <figcaption id="fig-quoted-asserted-triple-alt">
+      <figcaption id="fig-quoted-asserted-graph-alt">
         An RDF graph containing two triples where one of them contains the other as the subject;
         hence, that other triple is both quoted and asserted.
       </figcaption>
     </figure>
 
-    <p>Note that a <a>quoted triple</a> may also have a quoted triple as its subject or object.</p>
+    <p>A quoted graph may contain triples which also have quoted graphs as subjects or objects.</p>
 
-    <p class="issue" data-number="34">
-      This issue considers the potential for additional terminology related to <a>quoted triples</a>.
+    <p class="note">
+      The concept of a quoted graph is similar to that of a named graph,
+      however a quoted graph does not have a name, and is intended to be used as the
```

Comment:
Can a quoted graph *not* be named (identified), with a URI or other non-blank-node?

Reply:
A "quoted graph" (better - "graph term") does not have a name - i.e. there is not a attribute or characteristic of the graph that is its name. 

It can be named - that does not affect the graph.

(Anything can be given a name - and it does not change the thing so named.)